src/App.tsx

```diff
@@ -1,61 +1,128 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { useState } from 'react';
+
+const findUserById = (userId: number) => usersFromServer.find(u => u.id === userId);
+
+const todos = todosFromServer.map(todo => {
+  const user = findUserById(todo.userId);
+
+  return { ...todo, user };
+});
 
 export const App = () => {
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [title, setTitle] = useState('');
+  const [chosenUser, setChosenUser] = useState(0);
+  const [error, setError] = useState(false);
```

Comment:
```suggestion
  const [hasError, setHasError] = useState(false);
```
Boolean values should start from `is ``has`